djangocms_navigation/filters.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
+
+from cms.utils.i18n import get_language_tuple, get_site_language_from_request
+
+
+class LanguageFilter(admin.SimpleListFilter):
+    title = _("language")

```

Comment:
The language filter will need to be registered with the admin, an example of where this has been done before: https://github.com/FidelityInternational/djangocms-pageadmin/blob/master/djangocms_pageadmin/admin.py#L48